spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  let!(:my_user) {User.create(name: "James")}
+
+  it "GET index returns success" do
+    get :index
+    expect(response).to have_http_status(:success)
+  end
+
+  describe "POST create user" do
+
+      it "creates a new user successfully" do
+        post :create, user: {name: "james"}
+        expect(response).to have_http_status(201)
```

Comment:
This is a great start.  You probably want to check what is in the response though.  You want it to respond with JSON, so you might need to deserialize back to ruby objects it, like `JSON.parse(response.body)`